prow/getting_started.md

```diff
@@ -0,0 +1,38 @@
+# How to turn up a new cluster
+
+Prow should run anywhere that Kubernetes runs. Here are the steps required to
+set up a basic prow cluster on GKE.
```

Comment:
Maybe make GKE link to the GKE docs or something? It might also be useful to point out what `ZONE` etc are below by linking to some GCP docs.

Reply:
I don't think the doc should explain any GKE/GCP concepts, but pointing people to where they can find them seems like it might be helpful.

Reply:
I added a link to container engine docs.